src/lib/components/Editor/Editors/ArticleEditor.tsx

```diff
@@ -1,22 +1,34 @@
+import { Container } from '@material-ui/core';
 import React, { useEffect, useRef, useState } from 'react';
 import ReactQuill, { Quill } from 'react-quill';
+import BorderBottom from '../../Border';
+import ContentPreviewContainer from '../ContentPreviewContainer';
 import GeneralEditor from '../GeneralEditor';
 import ArticleEditorToolbar from './ArticleEditorToolbar';
 
 const ArticleEditor: React.FC = () => {
   const [editor, setEditor] = useState<Quill>();
```

Comment:
editor is not a state here

Reply:
seems like you just need a constant for that = articleEditor.current.getEditor()

Reply:
state is something that might me changed, the editor is static :)

Reply:
I have tried to use const instead of state but, `articleEditor` initially is null, and I have decided to use `useEfect` to set the `editor` after the component renders. 

Logs example - https://i.imgur.com/xsXZGZL.png